apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemDAO.java

```diff
@@ -584,6 +611,27 @@ public Mono<List<WorkspaceAndResourceId>> getDatasetItemWorkspace(@NonNull Set<U
         });
     }
 
+    @Override
+    public Flux<DatasetItemSummary> findDatasetItemSummaryByDatasetIds(Collection<UUID> datasetIds) {
```

Comment:
This help us the the trace of the method in New Relic

```suggestion
     @WithSpan
    public Flux<DatasetItemSummary> findDatasetItemSummaryByDatasetIds(Collection<UUID> datasetIds) {
```

Reply:
Updated